vanilla/vanilla
# Fix markdown quote code block

Closes https://github.com/vanilla/support/issues/1171

**Notes:**
- This changes markdown default syntax of code blocks from indented to fenced code.
- I had to override `transform()` which is not ideal, but it was the only way to get all the functions running in the right order.

**Steps to test:**
- Make sure you have "Makrdown" as your posting format
- Make a comment with some block html, for example:
```
<ul>
    <li>something</li>
    <li>something else</li>
    <li>another thing</li>
</ul>
```
- Quote that comment and observe the rendering of the quote is broken.
- Checkout this branch.

**Futher testing:**
- Some examples of content you can test to see if they render correctly:

```
> @Tester_McTesterson said:
> ~~~
> <ul>
>     <li>something</li>
>     <li>something else</li>
>     <li>another thing</li>
> </ul>
> ~~~
> 
>     <ul>
>         <li>something</li>
>         <li>something else</li>
>         <li>another thing</li>
>     </ul>
> 
> 
> ```
> <ul>
>     <li>something</li>
>     <li>something else</li>
>     <li>another thing</li>
> </ul>
> ```
> `<ul><li>something</li><li>something else</li><li>another thing</li></ul>`
> 
> 
>         public function addBreaks() {
>             $this->span_gamut = array_replace($this->span_gamut, [
>                 'doStrikeout' =>  15,
>                 'doSoftBreaks' => 80,
>             ]);
>         }
> 
```
a bunch of code
```
